### ISSUE TITLE
Single node backup fails

### ISSUE THREAD

**Issue by mviitane** [k0sproject/k0s#990]
**Version**
```
$ k0s version
v1.21.2+k0s.0
```
**Platform**
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```
**What happened?**
Backup fails when k0s has been started with '--single' option.
```
curl -sSLf https://get.k0s.sh | sudo sh
sudo k0s install controller --single
sudo k0s start
sudo k0s backup --save-path=.
INFO[2021-06-24 06:59:22] Backup step: etcd                            
Error: failed to create backup on step `etcd`: open /var/lib/k0s/pki/apiserver-etcd-client.crt: no such file or directory
2021-06-24 06:59:22.743027 I | failed to create backup on step `etcd`: open /var/lib/k0s/pki/apiserver-etcd-client.crt: no such file or directory
```

**Expected behavior**
Backup function shouldn't require etcd. Option '--single' starts SQLite datastore by default.

**Additional context**
Backup/restore documentation should be enhanced to cover SQLite in addition to etcd.
https://docs.k0sproject.io/main/backup/

**Comment by mviitane** [comment]
Workaround:
Manually configure kine as datastore in k0s.yaml
```
spec:
  storage:
    type: kine
```
Start the backup with the config file as an argument:
`sudo k0s backup --config k0s.yaml --save-path=.
`

**Comment by soider** [comment]
Should it be taken into consideration since we have CRD config merged?

**Comment by jnummelin** [comment]
Hmm, not sure but this might get fixed as part of #1247 

@trawler or was this one of the cases which couldn't be handled in the new config file linking?

**Comment by mviitane** [comment]
Seems to be fixed. Backup works fine also for a single node with kine.
